layouts/partials/example-python.html

```diff
@@ -9,24 +9,18 @@
 <pre><code class="no-copy" tabindex="-1">import pydgraph
 import json
 
-def query_data(client):
-
+def main():
+    client_stub = pydgraph.DgraphClientStub("localhost:9080")
```

Comment:
Seems like this is adding a client creation code to *every* example. Seems too verbose to me.